src/test/java/com/jcabi/xml/XSLDocumentTest.java

```diff
@@ -139,11 +141,20 @@ public void transformsIntoText() throws Exception {
      */
     @Test
     public void stripsXml() throws Exception {
+        final StringWriter out = new StringWriter();
+        final BufferedWriter writer = new BufferedWriter(out);
+        writer.write("<a>");
+        writer.newLine();
+        writer.write("<b/>");
+        writer.newLine();
+        writer.write("</a>");
+        writer.close();
+        final String actual = out.getBuffer().toString();
         MatcherAssert.assertThat(
             XSLDocument.STRIP.transform(
                 new XMLDocument("<a>   <b/>  </a>")
             ).toString(),
-            Matchers.containsString("<a>\n<b/>\n</a>")
+            Matchers.containsString(actual)
```

Comment:
maybe, it would be better to inline that ```actual``` following way:
```java
		Matchers.containsString(
				new StringBuilder()
						.append("<a>")
						.append(System.lineSeparator())
						.append("<b/>")
						.append(System.lineSeparator())
						.append("<a/>")
						.toString()
		);
```
what do you think?